config

```diff
@@ -588,5 +588,27 @@ else
     CORE_LIBS="$CORE_LIBS $ngx_module_libs"
 fi
 
+# ----------------------------------------
+
+if [ $PCRE != NO -a $PCRE != YES ]; then
```

Comment:
When dynamically linked, `$PCRE` will be `YES` or `NO`. If it is neither, it is statically linked. https://github.com/nginx/nginx/blob/master/auto/lib/pcre/conf#L6